### DISCUSSION TITLE
Simulator occasionally crash

### DISCUSSION THREAD

**Discussion by dotchen** [carla-simulator/carla#4639] [Q&A] [answered]
Hi CARLA team,

Thank you so much for your great work!

I am running the prebuilt CARLA 0.9.12 on ubuntu 18.04, and I found that the simulator occasionally crash with the following message:

```
Signal 11 caught.
Malloc Size=65538 LargeMemoryPoolOffset=65554
CommonUnixCrashHandler: Signal=11
Engine crash handling finished; re-raising signal 11 for the default handler. Good bye.
LowLevelFatalError [File:Unknown] [Line: 1214]
GameThread timed out waiting for RenderThread after 60.00 secs
```

I was launching the simulator using

```
./CarlaUE4.sh -RenderOffscreen -vulkan
```

I haven't encountered this when using 0.9.10.1 previously. Any suggestions on how to fix this? Thank you very much!

**Comment by HaechanMarkBong** [comment]
Hi dotchen, I have the same problem. Any chance you found a solution?

**Reply by tby-udel** [reply]
Hello, did you solve this problem

**Comment by xavisolesoft** [comment]
Could you check if any error log was generated in the next path, please?
```
~/.config/Epic/CarlaUnreal/Saved/Crashes/
```

**Reply by tby-udel** [reply, marked as answer]
I have solved the problem, mine is caused by GPU driver, I re-installed a proper driver for my GPU and the simulator started working perfectly.

**Reply by xavisolesoft** [reply]
Great!
